designer/apps/api/v1/serializers.py

```diff
@@ -4,55 +4,105 @@
 from designer.apps.pages.models import IndexPage, ProgramPage
 
 
-class BrandingField(serializers.PrimaryKeyRelatedField):
-    def to_representation(self, value):
+class PageSerializer(serializers.ModelSerializer):
+
+    class Meta:
+        model = Page
+        fields = (
+            'id',
+            'title',
+            'slug',
+            'last_published_at',
+        )
+
+
+class BrandedPageSerializerMixin(object):
+
+    def get_branding(self, obj):
+        """
+        Get serialized version of branding for the given obj
+        Args:
+            obj: (ModelSerializer) The branded page currently being serialized.
+
+        Returns:
+            (dict) Serialized version of the branding for the given obj
+        """
+        branding = obj.branding.first()
+
+        if not branding:
+            return {}
+
         ret = {
```

Comment:
nit: You could just return the object here rather than creating ret and then returning it.

Reply:
sounds good